lib/liquid_voting/voting.ex

```diff
@@ -143,6 +143,24 @@ defmodule LiquidVoting.Voting do
     |> Repo.preload([:participant])
   end
 
+  @doc """
+  Gets a single vote by participant id and proposal_url
+
+  ## Examples
+
+      iex> get_vote_by_participant_id("a6158b19-6bf6-4457-9d13-ef8b141611b4", "https://proposals.net/2")
+      => %Vote{}
+
+      iex> get_vote_by_participant_id("a6158b19-6bf6-4457-9d13-ef8b141611b4", "https://proposals.com/non-existant-proposal")
+      => nil
+  """
+  def get_vote_by_participant_id(participant_id, proposal_url) do
```

Comment:
Should also be constrained by organization_id

Reply:
I will do this immediately this PR is merged (see issue #164).